ruby/conclusion/project_ruby_final.md

```diff
@@ -22,9 +22,10 @@ This is a great project to have as a part of your portfolio going forward becaus
 </div>
 
 ### Additional resources
-This section contains helpful links to other content. It isn't required, so consider it supplemental.
 
-* [Illustrated rules of Chess](http://www.chessvariants.org/d.chess/chess.html)
-* [Chess notation for humans and computers](https://en.wikipedia.org/wiki/Chess_notation)
-* If you're feeling stuck, you should visit the community Discord!
-* You don't need this...  resist the temptation! Here is [a discussion of this problem from RubyQuiz](http://rubyquiz.com/quiz35.html). Be warned, this resource contains a blueprint for how to complete this project. That means, if you decide to peruse it, you won't get the opportunity to dig into this problem.
+This section contains helpful links to related content. It isn't required, so consider it supplemental.
```

Comment:
Every place you've copied and pasted this sentence from the Layout Style Guide has a small problem:
```
This section contains helpful links to related content. It isn't required, so consider it supplemental.
```
The `isn't` uses an incorrect apostrophe character and Markdownlint is complaining about it. I've put in a PR to fix this on the layout style guide here: #27319 

They also need to be fixed in this PR before merging.

Reply:
Well, this is awkward. I've copied and pasted from the layout style guide for **every** PR, which there are 7 of. 

I'll take time to fix all of the apostrophes in every PR, but in the future I'll definitely make issue before doing a big change like this as it wouldn't have happened had I made one!

Reply:
> Every place you've copied and pasted this sentence from the Layout Style Guide has a small problem:
> 
> ```
> This section contains helpful links to related content. It isn't required, so consider it supplemental.
> ```
> 
> The `isn't` uses an incorrect apostrophe character and Markdownlint is complaining about it. I've put in a PR to fix this on the layout style guide here: #27319
> 
> They also need to be fixed in this PR before merging.

I've converted all the ` to '. Reverting the 4 commits was tricky to do, since I've been using GitHub in the browser and hadn't had to clone the repository yet.